### ISSUE TITLE
Determine Libreoffice Still or stable.

### ISSUE THREAD

**Issue by Gil-s-rouen** [AdliceSoftware/UCheck#37]
### Preliminary Checks

- [X] I use a modern operating system (Windows 7 with SP1 or newer)
- [X] I checked that the feature I'm requesting is not available in UCheck Beta
- [X] I checked that the feature I'm requesting was not already requested

### Which feature do you want to be added ?

Hello,

I think that there is a detection fault for Libreoffice, on my workstation, it is the latest version 7.6.4 "Still" (Stable) that is installed and not the latest version 24.2.0 "Fresh". The Libreoffice "Still" version is technically up to date, and Ucheck should not offer the "Fresh" update, as it is a deliberate choice to install the "Still" version.
It is necessary to find a way to determine the type of "Still" or "Fresh" installation on the workstation before proposing the "Fresh" update, and idem for Libreoffice Help Pack, but ... Is there a way? For the moment, I haven't found one.

Best regards

### ucheck-report

Program       : UCheck
Version       : 5.2.0.0-BETA1
x64           : Yes
Program Date  : Dec 22 2023
Location      : C:\Users\******\Downloads\UCheck64.exe
Premium       : Yes
Company       : Adlice Software
Website       : https://www.adlice.com/
Contact       : https://adlice.com/contact/
Website       : https://adlice.com/download/ucheck/
Operating System : Windows 10 (10.0.19045) 64-bit
64-bit OS     : Yes
Startup       : 0
WindowsPE     : No
User          : *****************
User is Admin : No
Date          : 2024/02/12 08:28:24
Type          : Scan
Aborted       : No
Scan Mode     : Standard
Updates Count : 7

************************* Warnings *************************

************************* Updates *************************
LibreOffice 7.6.4.1 (64-bit), version 7.6.4.1
  [+] Available Version        : 24.2.0
  [+] Size                     : 1,39Â Go
  [+] Wow6432                  : No
  [+] Portable                 : No
  [+] update_location          : C:\Program Files\LibreOffice\

LibreOffice 7.6 Help Pack (French) (64-bit), version 7.6.4.1
  [+] Available Version        : 24.2.0.3
  [+] Size                     : 30,7Â Mo
  [+] Wow6432                  : No
  [+] Portable                 : No
  [+] update_location          : C:\Program Files\LibreOffice\

### Don't forget to vote !

- [x] Bugs and new features will be treated by priority and popularity.
Make sure **you vote below** with a :thumbsup: !

**Comment by TheAncient2** [comment]
This "Still" vs "Fresh" designation is probably the same thing that other software developers call "Production" vs "Beta".
A related thread can be found at https://forum.adlice.com/index.php?topic=3827.0.
According to that thread, UCheck is not supposed to propose updates to Beta versions and probably should not have proposed an update to a "fresh" version either.
Another related thread regarding erroneous detections is at https://forum.adlice.com/index.php?topic=3808.0
Maybe this request should be added there? 
I definitely support adding an option to include or exclude "Fresh" or "Beta" versions from the list of updates offered.

**Comment by Tigzy** [comment]
It's pretty confusing. They don't say on their website (https://www.libreoffice.org/download/download-libreoffice/) that version 24 is a Beta, so I don't believe it is. It's probably the most up to date, but not a LTS (7.6 is an LTS). 

If you look at Ubuntu for example: https://ubuntu.com/download/desktop
They propose 2 versions, 22 LTS and 23.10. None of them are BETA versions, just 2 branches with different progression. 
BETAS are usually called pre-release, release-candidate or a different term. 

For now I believe we'll stick to version 24 for LibreOffice, because there's no real proof it's a BETA, and also because it will confuse users already on this version with UCheck.

**Comment by Tigzy** [comment]
If found this page: https://www.libreoffice.org/download/pre-releases/

```
These are pre-release versions and are not recommended for production use. Are you interested in helping out? Please read the release notes and [visit our Testing page](https://www.libreoffice.org/community/qa/).

No RC is currently available for the 24.2 codeline
RC builds are in release configuration and will replace your existing LibreOffice installation!
No RC is currently available for the 7.6 codeline
RC builds are in release configuration and will replace your existing LibreOffice installation!
```

Also the way those 2 codelines are displayed and marketed on the download page makes me think it's not a BETA at all. They are on the same level, same color scheme and same size and importance in the page.

**Comment by Gil-s-rouen** [comment]
Hello,
I don't agree at all about the presence or absence of a beta version on the official download page whose address I gave in my post.
Version 24 (fresh) is a consumer version which changed version numbers at the beginning of the year from 7.6.x to 24.1.x and version 7.6.x (still or stable) which followed a slightly lower versioning which progressed more slowly a bit like Firefox has a consumer version and an "enterprise" version which is "esr" for which UCheck makes a good difference. On the same download page you can find a link to the Beta versions.
In short, like Firefox, Libreoffice has a "consumer" version and an "enterprise" version ESR or LTS, and the "enterprise" version is STILL.
Indeed, you say that a lot of people report version 24 because it's "consumer", so maybe there are fewer techs using UCheck.
I bought the "technician" version of UCheck and to be credible, UCheck must differentiate between the "consumer" and "enterprise" versions, as the late SUMo knew how to do. I had bought the "lifetime" version, and when SUMo disappeared, which was "THE" reference in terms of software version detection, I turned to UCheck, which seemed to me the most serious alternative to help me in my work as an IT asset manager. Perhaps I was mistaken.
Best regards.
